lib/b2b/oidc.ts

```diff
@@ -0,0 +1,55 @@
+import { BaseResponse, request, fetchConfig } from "../shared";
+import { OIDCConnection } from "./sso";
+
+export interface CreateOIDCConnectionRequest {
+  organization_id: string;
+  display_name?: string;
+}
+
+export interface CreateOIDCConnectionResponse extends BaseResponse {
+  connection: OIDCConnection;
+}
+
+export interface UpdateOIDCConnectionRequest {
+  organization_id: string;
+  connection_id: string;
+  display_name: string;
+  client_id: string;
+  client_secret: string;
+  issuer: string;
+  authorization_url: string;
+  token_url: string;
+  user_info_url: string;
+  jwks_url: string;
```

Comment:
All of these fields are marked as required, but only `organization_id` and `connection_id` are. Can we update?

Reply:
Updated